app.py

```diff
@@ -0,0 +1,5 @@
+from app import create_app
+
+if __name__ == "__main__":
+    app = create_app()
+    app.run(debug=True)

```

Comment:
Probably better to use an environment variable to control this.
https://flask.palletsprojects.com/en/2.1.x/quickstart/#debug-mode
```suggestion
    app.run()
```

@jeffbuswell is there some way we can enable debug for dev but disable it for prod?

Reply:
@aroxby-wayscript We want our customers to have a persistent DB. Since the DB is stored inside the Lair, this would only be possible with dev deployments. So, Jeff suggested we only run in dev.